### ISSUE TITLE
Changing the contentView+Layout of an existing FP controller doesn't update it's frame size

### ISSUE THREAD

**Issue by mlinsenbard** [SCENEE/FloatingPanel#77]
Hello! First off, thanks for all your work on this Framework.

I'm running into an issue where the frame of a FloatingPanel Controller isn't being updated when I change it's layout and contentView.

For example, I initialize a FPC with a specific contentVC and Layout and add it to a parent VC. I then later tap a button to swap out the FPC's contentVC and change its Layout and do something like:
```swift
self.floatingController.set(contentViewController: vc1)
self.activeLayout = VC1FloatingPanelLayout()
self.floatingController.updateLayout()
```
The Layout and contentVC changes happen successfully, but if the .full position inset of the original Layout is larger than the .full position inset of the new layout, you end up with a gap of that difference at the bottom of the FPC as shown in the image.

![simulator screen shot - iphone xs - 2018-12-05 at 15 29 08](https://user-images.githubusercontent.com/5933636/49552787-e16d5380-f8a9-11e8-86d6-d637514f9fd2.png)

All the viewControllers I'm using are made in Storyboards and use Autolayout/Safe Area Margins.

If you want more info or I'm doing something wrong, let me know!

**Comment by SCENEE** [comment]
Thanks for your report, @mlinsenbard. I'm sorry to be late. 

I'm going to release v1.3.0 soon. The .full position frame will be affected by the release. So I'd like to try it after the release and let me know the issue is still reproduce or not.

And if you could create a sample project to reproduce this issue, it would be much helpful for me.

**Comment by SCENEE** [comment]
This can be fixed by  #123. If you still have the same issue, Please feel free to reopen this.
